tutorial/consuming_packages/the_flexibility_of_conanfile_py.rst

```diff
@@ -190,9 +190,12 @@ platform without adding more changes.
 
         def requirements(self):
             self.requires("zlib/1.2.11")
+            if self.settings.os == "Windows":
```

Comment:
This needs to be reviewed by @czoido, cause this tutorial has always been a bit challenging to follow in sync the code and the docs

Reply:
This is in fact an update to the docs to sync them back :)